### DISCUSSION TITLE
Add badge to table element

### DISCUSSION THREAD

**Discussion by Ayabadsir** [orchidsoftware/platform#1919] [Q&A] [answered]
Hi, 

I tried this:

`TD::make('statut', __('Statut'))
                ->sort()
                ->cantHide()
                ->badge(function ($books) {
                    return $books->statut;
  }),`

And I got Method Orchid\Screen\TD::badge does not exist.

What can I do please!

**Comment by gruz** [comment, marked as answer]
You should use render method, smth. like this
```
            TD::make('name 'Name
                ->render(function (User $user {
                    return '<div class="badge bg-dark">'. $user->name , '</div>';
                }),
```

**Reply by Ayabadsir** [reply]
THANKS @gruz. IT'S WORKS!!!
